presidio-analyzer/presidio_analyzer/chunkers/chunking_utils.py

```diff
@@ -0,0 +1,102 @@
+"""Utility functions for processing text with chunking strategies."""
+from typing import Any, Callable, Dict, List
+
+from presidio_analyzer.chunkers.base_chunker import BaseTextChunker
+
+
+def predict_with_chunking(
```

Comment:
Could these methods be part of the base chunker? What are the advantages of having it as a utility? Are there cases where one would use those outside the scope of chunking?